src/script.js

```diff
@@ -1,18 +1,32 @@
-let myTimer;
+const long = 2;
+const medium = 1.5;
+const short = 1;
+const myTimer = 1000;
 
-var start = function () {
-  myTimer = setInterval(updater, 5000);
+let running = false;
+
+var start = async function () {
+  console.log('start');
+  running = true;
+  for (let i = 0; i <= 40 && running; i++) {
+    console.log('interval ', i);
+    const combo = updater();
+    await sleep(getMultiplier(combo) * myTimer);
+  }
 };
 
 var stop = function () {
-  clearInterval(myTimer);
+  running = false;
+  console.log('end');
 };
 
-var updater = function () {
-  var combo = getRandomNumber() + getRandomLetter();
-  var combos = document.getElementById('combination');
+let updater = function () {
```

Comment:
I don't think this name describes what the function does

Reply:
changed to `updateCombo`